[quantization] Refactoring names for consistency

z-a-f opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #27779 [quantization] Changing observer name
* **#27670 [quantization] Refactoring names for consistency**

Differential Revision: [D17846269](https://our.internmc.facebook.com/intern/diff/D17846269)

Commit:
[quantization] Refactoing names for consistency

Commit:
Update on "[quantization] Refactoing names for consistency"


Differential Revision: [D17846269](https://our.internmc.facebook.com/intern/diff/D17846269)

jerryzh168 reviewed (approved):
this reminds me that we need to rename 'self.observer' to 'self.activation_post_process' do we want to do it now?

jianyuh reviewed (approved):
LGTM!

zafartahirov commented:
> this reminds me that we need to rename 'self.observer' to 'self.activation_post_process' do we want to do it now?

Might as well

zafartahirov commented:
> this reminds me that we need to rename 'self.observer' to 'self.activation_post_process' do we want to do it now?

Wait, in which context?

Commit:
Update on "[quantization] Refactoing names for consistency"


Differential Revision: [D17846269](https://our.internmc.facebook.com/intern/diff/D17846269)

zafartahirov commented:
> this reminds me that we need to rename 'self.observer' to 'self.activation_post_process' do we want to do it now?

Created a new PR: #27779

Commit:
Update on "[quantization] Refactoing names for consistency"


Differential Revision: [D17846269](https://our.internmc.facebook.com/intern/diff/D17846269)

Commit:
Update on "[quantization] Refactoring names for consistency"


Differential Revision: [D17846269](https://our.internmc.facebook.com/intern/diff/D17846269)

zou3519 commented on docs/source/quantization.rst:
```diff
@@ -190,10 +190,10 @@ Layers for the quantization-aware training
-    * :attr:`~torch.quantization.default_qat_qconfig` — Same as ``QConfig(activation=default_fake_quant, weight=default_weight_fake_quant)`` (See :class:`~torch.quantization.QConfig.QConfig`)
-    * :attr:`~torch.quantization.default_dynamic_qconfig` — Same as ``QConfigDynamic(weight=default_weight_observer)`` (See :class:`~torch.quantization.QConfig.QConfigDynamic`)
-    * :attr:`~torch.quantization.float16_dynamic_qconfig` — Same as ``QConfigDynamic(weight=NoopObserver.with_args(dtype=torch.float16))`` (See :class:`~torch.quantization.QConfig.QConfigDynamic`)
+    * :attr:`~torch.quantization.default_qconfig` — Same as ``QConfig(activation=default_observer, weight=default_weight_observer)`` (See :class:`~torch.quantization.qconfig.QConfig`)
```
`default_qconfig` doesn't exist as an attribute, so it is better to label it as ``default_qconfig``
![image](https://user-images.githubusercontent.com/5652049/66953340-5235de00-f02c-11e9-92c5-6719ccca7ff3.png)

z-a-f replied:
I will send a PR to set it as just verbatim text.
